docs/examples/BasicMulti.js

```diff
@@ -8,6 +8,14 @@ export default () => (
     defaultValue={[colourOptions[2], colourOptions[3]]}
     isMulti
     name="colors"
+    onValueClick={onValueClick}
```

Comment:
I thought we decided to leave this out of the basic implementation?

Reply:
fixed!